### PR TITLE
summary for Understanding Browser Rendering

### PR DESCRIPTION
**Pull request by Chimdiya1**
# What Does This PR Do?
- Adds summary for the Understanding browser rendering session

## Github Issue Referenced
- https://github.com/segunolalive/frontend-weekly/issues/48

## Link To Session on [Frontend Weekly](https://frontendweekly.dev/)
- [Understanding Browser Rendering](https://frontendweekly.dev/sessions/understanding-browser-rendering)

fixes: #48

### CODE REVIEW

**Comment by Chimdiya1** [comment]
@segunolalive

File: src/session-summaries/understanding-browser-rendering.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+summarized_by: Chimdia Anyiam
+summarized_by_social_media: https://twitter.com/AnyiamChimdia
+
+---
+
+# Understanding browser rendering
+
+This session has Enodi Audu introducing the concept of browser rendering and steps that the browsers take to convert our HTML code to what we can see on the screen.
+
+The browser receives raw bytes of data that it does not understand, these bytes are then converted to characters depending on the character encoding specified in the HTML head tag e.g. UTF-8. These characters are tokenized and the tokens are converted to nodes which are likned in tree data structure known as the Document Object Model (DOM). The host does well too explain this in more details.
```
Comment (segunolalive) [inline src/session-summaries/understanding-browser-rendering.md:14]: This is concise and clear. Thanks.

Can you change the last sentence from _The host does well `too`_ to _Enodi does well `to`_

File: src/session-summaries/understanding-browser-rendering.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+What you’ll learn:
```
Comment (segunolalive) [inline src/session-summaries/understanding-browser-rendering.md:16]: Make this a header.

## What you’ll learn:

**Review by segunolalive — CHANGES_REQUESTED** [review]
This is good.

Just make the suggested changes and we done.

**Comment by Chimdiya1** [comment]
done @segunolalive

**Review by segunolalive — APPROVED** [review]

**Pull request merged**